### ISSUE TITLE
Labels on GeoJSON "display name" and "identifier" property settings are reversed

### ISSUE THREAD

**Issue by thucnc** [metabase/metabase#3537]
I got error as below attachment when trying to import a geojson file. (I try several files but got the same error) Here is my file url: http://wikisend.com/download/882322/ho-chi-minh-city_vietnam_admin.geojson
![screen shot 2016-10-12 at 11 07 50 am](https://cloud.githubusercontent.com/assets/2088055/19297163/598e20a4-906c-11e6-99b0-dbd260a6d2e1.png)

Thanks,

**Comment by camsaul** [comment]
@thuncnc when I click that link I get a website, not a JSON file.

**Comment by tlrobinson** [comment]
@thucnc I've confirmed if you download that file and upload it to somewhere that serves the raw file it will work fine file (https://gist.github.com is nice because it shows a preview of GeoJSON files):

![screenshot 2016-10-11 22 47 47](https://cloud.githubusercontent.com/assets/18193/19298746/e873bb48-9004-11e6-850a-7c9078a17d9a.png)

![screenshot 2016-10-11 22 48 18](https://cloud.githubusercontent.com/assets/18193/19298747/e873d3f8-9004-11e6-91d4-de04a568b650.png)

**Comment by thucnc** [comment]
Thank you @tlrobinson, but I still unable to build a choropleth map with our imported geojson. May you advise or give any reference ?

**Comment by tlrobinson** [comment]
@thucnc Where are you stuck? Are you able to see the outline of the regions like in my second screenshot?

After that you will need a table that can be grouped by the same values as the "region’s identifier" property you selected when adding the GeoJSON, for example if you selected "name" your data would have to have a column containing values like "Quận 2".

I don't have a table like that but you can fake it with a `UNION` SQL query for testing purposes: 

```
select 'Quận 2' as name, 2 as value
union
select 'Quận 9' as name, 9 as value
union
select 'Thị xã Dĩ An' as name, 20 as value;
```

![screenshot 2016-10-12 00 32 02](https://cloud.githubusercontent.com/assets/18193/19301293/8e61ba2e-9013-11e6-872a-13defce123d7.png)

**Comment by thucnc** [comment]
@tlrobinson thank you for your advice. After the same property for both identifier and display name of regions, I can get the choropleth now. Is it a bug or a feature ?

**Comment by tlrobinson** [comment]
Oh you're right, there is a bug, the settings are reversed! We'll fix this in 0.20.1, but for now just pretend "Which property specifies the region’s display name?" and "Which property specifies the region’s identifier?" are swapped.

**Comment by camsaul** [comment]
Fixed by #3542
